### ISSUE TITLE
how do you get pretrained model

### ISSUE THREAD

**Issue by yuyang16101066** [wjn922/ReferFormer#19]
Thanks for your great work. I would like to know how you get pretrained model, like video_swin_tiny_pretrained.pth. In my understanding, it's different from Joint training with Ref-COCO/+/g datasets.

**Comment by wjn922** [comment]
Hi, it is indeed the pretrained model is different joint training model. The pretrained models are only trained using Ref-COCO/+/g datasets in a image level (setting num_frmaes=1).

**Comment by nero1342** [comment]
Hi, did you use the pretrained models on RefCOCO/+/g datasets for the joint model? And is it imbalance when joint training between 3M expressions from RefCOCO/+/g and 13k expressions from RefYTVOS?

**Comment by wjn922** [comment]
We do not use the pretrained model for joint trainnig. We do not adopt the balance sampling of RefCOCO/+/g and RefYTVOS, though their scales are different.

**Comment by zhenghao977** [comment]
@wjn922 Thanks for your great work! I would like to know epochs/learning rate/lr_drop out for getting the pretrained model on Ref-COCO

**Comment by wjn922** [comment]
@zhenghao977 We use 32 V100 GPUs for the pretrained models. The total epoch is 12 and lr drops at the 8th and 10th epoch. The learning rate keeps the same as default setting, and batch size is set as 2 in each gpu. Please refer to https://github.com/wjn922/ReferFormer/issues/7 for the pretraining script.
